pages/home.tsx

```diff
@@ -1,9 +1,14 @@
-import React from "react";
+import React, { useContext } from "react";
 import type { NextApplicationPage } from "./_app";
-//This is the page that is rendered when the 'Home' button from the Navbar is clicked
+import { AuthContext } from "../context/AuthContext";
+import { AdminCalendar } from "../components/Calendar/AdminCalendar";
+import { UserCalendar } from "../components/Calendar/UserCalendar";
 
+//This is the page that is rendered when the 'Home' button from the Navbar is clicked
 const Home: NextApplicationPage = () => {
-  return <div>Home</div>;
+  const { user } = useContext(AuthContext);
+
+  return user?.role == "Admin" ? <AdminCalendar /> : <UserCalendar userId={user?.id} />;
```

Comment:
Here, can we create 2 different components called AdminHomePage and TecherHomePage? Put the calendars inside there and render the *HomePage components instead of the Calendar's themselves